### PR TITLE
Allow multiple library YAML files. Add EI.yaml

### PR DESCRIPTION
**Pull request by nsoranzo**
Also: remove unused `fix-permissions.sh`

### CODE REVIEW

File: test.sh [hunk 1]
```diff
@@ -2,38 +2,36 @@
...
-		echo "ERROR: description is not correct ($lib_description != Galaxy Training Network Material)"
-		ec=1
-	fi
-
-	if [[ "$lib_can_user_add" != "true" ]]; then
-		echo "ERROR: user cannot add items to this library"
-		ec=1
-	fi
-
+for line in $(cat secrets.json | jq -c '.[]'); do
```
Comment (hexylena) [inline test.sh:5]: This is nicer! Nice find.

File: test.sh [hunk 2]
```diff
@@ -2,38 +2,36 @@
...
-		echo "ERROR: name is not correct ($lib_name != GTN - Material)"
-		ec=1
-	fi
-
-	if [[ "$lib_description" != "Galaxy Training Network Material" ]]; then
-		echo "ERROR: description is not correct ($lib_description != Galaxy Training Network Material)"
-		ec=1
-	fi
-
-	if [[ "$lib_can_user_add" != "true" ]]; then
```
Comment (hexylena) [inline test.sh:32]: Why did you replace `[[` with `[`? 

I have a strong preference for `[[`, it generally avoids a number of gotchas with `[`

**Review by hexylena — COMMENTED** [review]
Hi @nsoranzo, thanks for this contribution! The linewise parsing of the json is definitely a nice improvement and much safer.

- Would you mind changing the whitespace back to how it was? 
- Could you please re-add fix-permissions? It was needed at one point and I'd rather not lose it. I had been under the impression we ran it regularly in Jenkins but I'm not finding it currently, and I don't remember that issue being fixed.

File: test.sh (hunk 2 above)
Comment (nsoranzo) [reply test.sh:32]: Not of a fan of bashisms :D 

But I can revert.

Comment (hexylena) [reply test.sh:32]: No, it's ok, leave this! Just wanted to know the reasoning behind it, avoiding bashism and portability is a noble goal.

**Comment by nsoranzo** [comment]
> Would you mind changing the whitespace back to how it was?

Sure (but I hate that copy-pasting tab-separated shell code to a terminal is broken...)

> Could you please re-add fix-permissions? It was needed at one point and I'd rather not lose it. I had been under the impression we ran it regularly in Jenkins but I'm not finding it currently, and I don't remember that issue being fixed.

The exact same code is in run.sh, so it's not really lost?

**Comment by hexylena** [comment]
I've pushed a change updating secrets.json to use the new structure. When I have a minute in the future I'll extract the API keys into a separate file so this one can be stored in plain text, it'd be easier for everyone.

**Comment by hexylena** [comment]
> but I hate that copy-pasting tab-separated shell code to a terminal is broken

it is? I don't experience that at all. Neither when copying from GH or from a text editor, it pastes the raw tab character in and everything is happy.

> The exact same code is in run.sh, so it's not really lost?

ah, indeed, ok, indeed. Nevermind then.

**Comment by nsoranzo** [comment]
> it is? I don't experience that at all. Neither when copying from GH or from a text editor, it pastes the raw tab character in and everything is happy.

With one tab yes, but at the first double tab it stops with:
```
Display all 3352 possibilities? (y or n)
```
That's inside a GNOME Terminal.

**Comment by hexylena** [comment]
Fascinating! I'd never known people experienced that issue, it works properly for me in alacritty and xfce (the only two installed here) but I also rarely copy and paste scripts I'm working on. 

I guess I can't say no to being more accessible, so, leave the spaces.

**Comment by nsoranzo** [comment]
Thanks for the merge!

**Comment by nsoranzo** [comment]
Can you please start a build on https://build.galaxyproject.eu/job/usegalaxy-eu/job/shared-data/ when you have a sec?

**Comment by hexylena** [comment]
https://build.galaxyproject.eu/job/usegalaxy-eu/job/shared-data/94/console

File: EI.yaml [hunk 3]
```diff
@@ -0,0 +1,132 @@
+destination:
+  description: Earlham Institute Training Material
+  name: EI Training Material
```
Comment (nsoranzo) [inline EI.yaml:3]: @bgruening  Wondering if I should have used "Galaxy courses" instead, to not pollute the top-level libraries?

Comment (bgruening) [reply EI.yaml:3]: Yes, that would be better imho!
Can you change that?

**Comment by bgruening** [comment]
Thanks @nsoranzo!

File: EI.yaml (hunk 3 above)
Comment (nsoranzo) [reply EI.yaml:3]: Sure, https://github.com/usegalaxy-eu/shared-data/pull/9

**Pull request merged**